src/examples/drawer/DrawerDefault.tsx

```diff
@@ -21,7 +21,7 @@ const Example = () => {
       <Col textAlign="center">
         <Button onClick={open}>Open Drawer</Button>
         <DrawerModal isOpen={isOpen} onClose={close}>
-          <DrawerModal.Header>Tending a garden</DrawerModal.Header>
+          <DrawerModal.Header tag="h2">Tending a garden</DrawerModal.Header>
```

Comment:
Was there a specific reason for using a level 2 heading?

Reply:
It's what Level Access recommended in their remediation notes. (I imagine this was their recommendation because it's generally considered bad practice, from an a11y perspective, to have more than one h1 on a page.)